src/SAWScript/Crucible/LLVM/ResolveSetupValue.hs

```diff
@@ -259,19 +259,20 @@ typeOfSetupValue' cc env nameEnv val =
 resolveSetupVal :: forall arch.
   Crucible.HasPtrWidth (Crucible.ArchWidth arch) =>
   LLVMCrucibleContext arch ->
+  Crucible.MemImpl Sym ->
```

Comment:
The `MemImpl` can be looked up in the `LLVMCrucibleContext`, like so:
```haskell
            let mvar = Crucible.llvmMemVar (cc^.ccLLVMContext)
            mem0 <- case Crucible.lookupGlobal mvar globals of
              Nothing   -> fail "internal error: LLVM Memory global not found"
              Just mem0 -> return mem0
```
It's not pretty, but it reduces the possibility of the variable `mem` getting out of sync with `cc`.